package.json

```diff
@@ -17,6 +17,7 @@
   ],
   "scripts": {
     "build": "npm run util:copy-icons && stencil build && npm run util:add-custom-elements-index-mjs",
+    "build:watch": "npm run util:copy-icons && stencil build --watch && npm run util:add-custom-elements-index-mjs",
```

Comment:
Are there other use cases besides the npm link usage? If not, maybe the name could reflect this?

Reply:
Not that I can imagine right now.  It's only useful to me right now for rebuilding on changes for use in external apps.  I suppose it could also be used when you're just trying to serve out calcite `dist` with a generic web server running locally as well.

Reply:
Kind of like a `npm start:prod` mode perhaps...

Reply:
`build:watch` is clearer to me, `start:prod` infers you're passing the `--prod` flag to the stencil compiler

Reply:
Good point. I was just wondering if there was a way we could make it clearer when you would need to use that script. This could be simplified if we could bundle the icon assets differently and have the custom elements output properly output the mjs file. Then it'd just be a matter of passing flags to the stencil build script.

![nothing-to-see](https://user-images.githubusercontent.com/197440/100164578-ffed0080-2e6c-11eb-9dfa-19b7afe21e1f.gif)

Reply:
I like the sound of that.  Since I'm not as familiar with how those other scripts work, feel free to suggest a better way.  I'd love to better understand the purpose of each of these tasks too and why they're needed.